solr_config/schema.xml

```diff
@@ -180,18 +179,16 @@
   <field name="merged_rsID" type="text_general" indexed="true" stored="true" multiValued="false" />
 
   <!-- specific fields for Gene -->
-  <!-- <field name="reported_trait" type="text_general" indexed="true" stored="true" multiValued="false" /> -->
-  <field name="geneDescription" type="text_general" indexed="false" stored="true" multiValued="false" />
-  <field name="ensemblID" type="string" indexed="true" stored="true" multiValued="false" />
-  <field name="geneName" type="string" indexed="true" stored="true" multiValued="false" />
-  <field name="synonymsGene" type="text_general" indexed="true" stored="true" multiValued="false" />  
+  <field name="ensemblDescription" type="string" indexed="true" stored="true" multiValued="false" />
+  <field name="entrez_id" type="string" indexed="true" stored="true" multiValued="false" />
+  <field name="cytobands" type="text_general" indexed="true" stored="true" multiValued="false" />  
   <field name="biotype" type="text_general" indexed="true" stored="true" multiValued="false" />
-  <field name="variants" type="string" indexed="true" stored="true" multiValued="true" />
-
-  <field name="mapping" type="text_general" indexed="true" stored="true" multiValued="true" />  
-  <field name="mapping.chr" type="text_general" indexed="true" stored="true" multiValued="false" />  
-  <field name="mapping.start" type="int" indexed="true" stored="true" multiValued="false" />  
-  <field name="mapping.end" type="int" indexed="true" stored="true" multiValued="false" />  
+  <field name="rsIDs" type="string" indexed="true" stored="true" multiValued="true" />
+  <field name="synonymsGene" type="text_general" indexed="true" stored="true" multiValued="false" />
```

Comment:
Are "synonymsGene" intended to be searchable and displayed on the site? Currently gene synonyms can not be used in search and are not displayed on the site. I'd suggest removing this from the `schema.xml` document AND also the script that collects these.

Reply:
I don't want to remove that at this point. I'll experiment with different setting to make sure we are still returning gene for their synonyms wile not polluting the search results.

Reply:
Is this additional work requested or an experiementation?